changelog.md

```diff
@@ -14,21 +15,24 @@
 - bpk-component-banner-alert: 1.3.8 => 1.4.0
   - New `dismissable` configuration, see https://backpack.github.io/components/web/banner-alerts
 
-- react-native-bpk-component-card:
+- react-native-bpk-component-card: 1.1.6 => 1.1.7
```

Comment:
This was missed in the last release (also should have been a minor)

Reply:
In my defense there wasn't an entry in the changelog

Reply:
No defense required! :)

Reply:
In fact you are correct and I erroneously assumed it was released! Fixed.